src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanelButtons.razor

```diff
@@ -5,7 +5,7 @@
     <div class="c-spacer"></div>
     <div class="c-content">
         <div class="c-logo" @onclick="@OnLogoClick">
-            <img draggable="false" class="w-8 h-8 fill-current" alt="Actual Chat" src="/favicon.svg"/>
+            <logo-button-svg />
```

Comment:
I think it's useless to create web-components for just displaying SVG image. Just use img tag for that purpose.